### PR TITLE
Student Quiz Report Template and a few other small changes

### PR DESCRIPTION
**Pull request by pritamps**

### CODE REVIEW

File: app/models/student_quiz_report.py [hunk 1]
```diff
@@ -84,6 +62,9 @@ def get_student_quiz_report(self, student_id, quiz_id):
         student_id_quiz_id = student_id + "-" + quiz_id
         return self.__student_quiz_reports_db.get_student_quiz_report(student_id_quiz_id=student_id_quiz_id)
 
+    def get_student_quiz_report(self, report_id):
```
Comment (suryabulusu) [inline app/models/student_quiz_report.py:65]: two functions with same name `get_student_quiz_report` .. maybe name them as `get_report_by_id` , `get_report_by_student_quiz_id`?

File: app/routers/reports.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        api_router = APIRouter(prefix='/reports', tags=['reports'])
+        
+        @api_router.get('/')
+        def index_route():
+            return 'Welcome to all reports Index'
+
+        @api_router.get('/student_quiz_report/{report_id}')
+        def student_quiz_report(request: Request, report_id: str):
+            if report_id == None:
+                raise HTTPException(status_code=400, detail='Both student_id annd quiz_id have to be specified')
```
Comment (suryabulusu) [inline app/routers/reports.py:27]: `detail='report_id should contain both student_id and quiz_id'`

File: app/routers/reports.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    @property
+    def router(self):
+        api_router = APIRouter(prefix='/reports', tags=['reports'])
+        
+        @api_router.get('/')
+        def index_route():
+            return 'Welcome to all reports Index'
+
+        @api_router.get('/student_quiz_report/{report_id}')
```
Comment (suryabulusu) [inline app/routers/reports.py:24]: we can avoid `/student_quiz_report/` here? the route could be `/reports/{report_id}` ?

File: app/main.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 from models.student_quiz_report import StudentQuizReportController
 from db.student_quiz_reports_db import StudentQuizReportsDB
 from routers.student_quiz_reports import StudentQuizReportsRouter
+from routers.reports import ReportsRouter
 
 app = FastAPI()
```
Comment (suryabulusu) [inline app/main.py:12]: have to mount static to use in templates -- with -- `app.mount("/static", StaticFiles(directory="app/static"), name="static")
`

Comment (suryabulusu) [reply app/main.py:12]: also import `from fastapi.staticfiles import StaticFiles`

Comment (pritamps) [reply app/main.py:12]: Oh disappared from `git` merge. Will add!

File: app/models/student_quiz_report.py (hunk 1 above)
Comment (pritamps) [reply app/models/student_quiz_report.py:65]: It's just overloaded. I think it's okay. You think it's confusing kya?

File: app/routers/reports.py (hunk 3 above)
Comment (pritamps) [reply app/routers/reports.py:24]: No. Other report types will come soon. This class will handle all

File: app/routers/reports.py (hunk 2 above)
Comment (pritamps) [reply app/routers/reports.py:27]: Why?

File: app/models/student_quiz_report.py (hunk 1 above)
Comment (suryabulusu) [reply app/models/student_quiz_report.py:62]: is it allowed? i got an error while trying to call that function ; function takes 2 args but 3 were provided (when i sent quiz id and student id). https://stackoverflow.com/a/34546826/10985650

Comment (pritamps) [reply app/models/student_quiz_report.py:62]: Uh oh. Will change :)

File: app/static/style.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
+html {
```
Comment (sudheshna-donthineni) [inline app/static/style.css:1]: Just curious about one thing. Is this the only way to include CSS in FastAPI? The reason I'm asking, is when we start adding components and making the app bigger, having one CSS file could be a lot of work maintaining, no?

**Comment by sudheshna-donthineni** [comment]
Sorry, should have reviewed the previous PR as this comment fits there; in the routers directory `student_quiz_reports.py`, `StudentQuizReportModel` is not being accessed. Maybe we can remove it. Doesn't seem to be removed in this PR.

File: README.md [hunk 6]
```diff
@@ -27,6 +27,8 @@ This will create the the `student_quiz_reports` table.
 ## Accessing things
```
Comment (sudheshna-donthineni) [inline README.md:27]: If you read point 3, there is a typo before `app`.

Comment (pritamps) [reply README.md:27]: Fixed!

File: app/static/style.css (hunk 5 above)
Comment (pritamps) [reply app/static/style.css:1]: No no. We can have as many CSS files as we want.

**Comment by pritamps** [comment]
> 

Good point. Removed!

**Pull request closed without merging**